apps/framework-cli/src/cli/routines/flow.rs

```diff
@@ -29,9 +28,7 @@ impl FlowFileBuilder {
 
         let flow_file_path = project
             .flows_dir()
-            .join(source)
-            .join(destination)
-            .join(TS_FLOW_FILE);
+            .join(format!("{}__{}.ts", source, destination));
```

Comment:
Double underscore just in case the user wants `_` in the data model name.
Though both TS interfaces and and Python classes are supposed to be PascalCase.